### DISCUSSION TITLE
Electrs installed but not running.

### DISCUSSION THREAD

**Discussion by Damian101101** [raspiblitz/raspiblitz#4514] [Q&A]
Just reflashed a raspiblitz and re downloaded the chain.  Installed electrs and they won't run.  When I check sudo journalctl -u electrs  I get this:   

Apr 03 00:08:21 raspberrypi systemd[1]: Started Electrs.
Apr 03 00:08:21 raspberrypi electrs[1917288]: Error: please use `log_filters` to set logging verbosity
Apr 03 00:08:21 raspberrypi systemd[1]: electrs.service: Main process exited, code=exited, status=1/FAILURE
Apr 03 00:08:21 raspberrypi systemd[1]: electrs.service: Failed with result 'exit-code'.
Apr 03 00:09:21 raspberrypi systemd[1]: electrs.service: Scheduled restart job, restart counter is at 5.
Apr 03 00:09:21 raspberrypi systemd[1]: Stopped Electrs.
Apr 03 00:09:21 raspberrypi systemd[1]: Started Electrs.
Apr 03 00:09:21 raspberrypi electrs[1923919]: Error: please use `log_filters` to set logging verbosity
Apr 03 00:09:21 raspberrypi systemd[1]: electrs.service: Main process exited, code=exited, status=1/FAILURE
Apr 03 00:09:21 raspberrypi systemd[1]: electrs.service: Failed with result 'exit-code'.
Apr 03 00:10:21 raspberrypi systemd[1]: electrs.service: Scheduled restart job, restart counter is at 6.
Apr 03 00:10:21 raspberrypi systemd[1]: Stopped Electrs.
Apr 03 00:10:21 raspberrypi systemd[1]: Started Electrs.
Apr 03 00:10:21 raspberrypi electrs[1930235]: Error: please use `log_filters` to set logging verbosity
Apr 03 00:10:21 raspberrypi systemd[1]: electrs.service: Main process exited, code=exited, status=1/FAILURE
Apr 03 00:10:21 raspberrypi systemd[1]: electrs.service: Failed with result 'exit-code'.
Apr 03 00:11:21 raspberrypi systemd[1]: electrs.service: Scheduled restart job, restart counter is at 7.
Apr 03 00:11:21 raspberrypi systemd[1]: Stopped Electrs.

.... on a continual loop.

Any ideas?

**Comment by therealsupermax** [comment]
I reinstalled ElectrumRS and took about two weeks to complete. It finally began to run properly, but not sure why it took so long (usually takes about *up to* a few days, not two weeks+.
